doc/devdocs/modules/launcher/plugins/registry.md

```diff
@@ -0,0 +1,97 @@
+# Registry Plugin
+
+The registry plugin allows users to search the Windows registry.
+
+## Special functions (differ from the regular functions)
+
+* Support full base keys and short base keys (e.g. `HKLM` for `HKEY_LOCALE_MACHINE`).
+* Show count of sub-keys and count of values in the second result line.
+* Search for value names and values inside a registry key (syntax: `[RegistryKey]\\[ValueName]` and `[RegistryKey]\\[Value]`)
+
+## The Windows Registry
+
+The registry contains all settings for the Windows operating system and many settings of the installed (Windows only) programs.
+
+*Note: Linux and macOS program ports typical store the settings in it's own configuration files and not in the Windows registry.*
+
+For more information about the Windows registry, see [the official documentation](https://docs.microsoft.com/en-us/windows/win32/sysinfo/registry).
+
+For advanced information about the Windows registry, see [Windows registry information for advanced users](https://docs.microsoft.com/en-us/troubleshoot/windows-server/performance/windows-registry-advanced-users).
+
+## Score
+
+The score is currently not set on the results.
+
+## Important for developers
+
+### General
+
+* The assembly name is cached into `_assemblyName` (to avoid to many calls of `Assembly.GetExecutingAssembly()`)
+
+### Results
+
+* All results override the visible search result via `QueryTextDisplay` to avoid problems with short registry base keys (e.g. `HKLM`).
+* The most `ToolTipData` use tabs (`\t`) and new lines (`\n`).
+* The maximum length of a `Title` is set to 45 characters and will automatic truncated on the left side.
+
+## Microsoft.Plugin.Registry project
+
+### Important plugin values (meta-data)
+
+| Name            | Value                              |
+| --------------- | ---------------------------------- |
+| ActionKeyword   | `*`                                |
+| ExecuteFileName | `Microsoft.Plugin.Registry.dll`    |
+| ID              | `303417D927BF4C97BCFFC78A123BE0C8` |
```

Comment:
Add a pattern for this. It'll be similar to the other pattern, but it's why the checker isn't happy.

```
^\s+\|\s+ID\s+\|\s*\`[0-9A-F]{32}\`
```